includes/migration-guide/retargeting/winforms/accessibility-improvements-windows-forms-controls.md

```diff
@@ -1,10 +1,8 @@
ed by the OS.</li><li><xref:System.Windows.Forms.ToolStripMenuItem> controls with their <xref:System.Windows.Forms.ToolStripMenuItem.Checked> property set to **true** now render the associated check mark in a contrasting system color.  Previously the check mark color was not contrasting enough and not visible in High Contrast themes.</li></ul>NOTE: Windows 10 has changed values for some high contrast system colors. Windows Forms Framework is based on the Win32 framework. For the best experience, run on the latest version of Windows and opt in to the latest OS changes by adding an app.manifest file in a test application and uncommenting the following code:<pre><code class="lang-xml">&lt;!-- Windows 10 --&gt;&#13;&#10;&lt;supportedOS Id=&quot;{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}&quot; /&gt;&#13;&#10;</code></pre></br>**Improved keyboard navigation**<ul><li>When a <xref:System.Windows.Forms.ComboBox> control has its <xref:System.Windows.Forms.ComboBox.DropDownStyle> property set to <xref:System.Windows.Forms.ComboBoxStyle.DropDownList?displayProperty=nameWithType> and is the first control in the tab order on the form, it now displays a focus rectangle when the parent form is opened using the keyboard. Before this change, keyboard focus was on this control, but a focus indicator was not rendered.</li></ul>**Improved Narrator support**<ul><li>The <xref:System.Windows.Forms.MonthCalendar> control has added support for assistive technologies to access the control, including the ability for Narrator to read the value of the control when previously it could not.</li><li>The <xref:System.Windows.Forms.CheckedListBox> control now notifies Narrator when a <xref:System.Windows.Forms.CheckBox.CheckState?displayProperty=nameWithType> property has been changed. Previously, Narrator did not receive notification and as a result users would not be informed that the <xref:System.Windows.Forms.CheckBox.CheckState> property had been updated.</li><li>The <xref:System.Windows.Forms.LinkLabel> control has changed the way it notifies Narrator of the text of in the control. Previously, Narrator announced this text twice and read &quot;&amp;&quot; symbols as real text even though they are not visible to a user. The duplicated text was removed from the Narrator announcements, as well as unnecessary &quot;&amp;&quot; symbols.</li><li>The <xref:System.Windows.Forms.DataGridViewCell> control types now correctly report the read-only status to Narrator and other assistive technologies.</li><li>Narrator is now able to read the System Menu of child windows in [Multiple-Document Interface]~/docs/framework/winforms/advanced/multiple-document-interface-mdi-applications.md) applications.</li><li>Narrator is now able to read <xref:System.Windows.Forms.ToolStripMenuItem> controls with a <xref:System.Windows.Forms.ToolStripItem.Enabled?displayProperty=nameWithType> property set to **false**. Previously, Narrator was unable to focus on disabled menu items to read the content.</li></ul> |
```

Comment:
not related your PR: 
This part causes an extraneous invalid tag to be printed on the screen:
`&lt;/p/&gt;<p/>`

![image](https://user-images.githubusercontent.com/12971179/47326124-2439db80-d61c-11e8-86a6-846969daf450.png)